cumulusci/tasks/preflight/recordtypes.py

```diff
@@ -0,0 +1,15 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from collections import defaultdict
+
+
+class CheckSObjectRecordTypes(BaseSalesforceApiTask):
+    def _run_task(self):
+        rts = defaultdict(list)
+        records = self.tooling.query("Select SobjectType, FullName FROM RecordType")[
```

Comment:
This should use query_all instead of query, or it won't include all results when there are many.

Reply:
Oof, big code review miss on my part. I'll throw up a quick PR to fix this.